jdaviz/configs/specviz/plugins/parsers.py

```diff
@@ -40,6 +40,8 @@ def specviz_spectrum1d_parser(app, data, data_label=None, format=None, show_in_v
         data_label = [data_label]
     elif isinstance(data, SpectrumList):
         pass
+    elif isinstance(data, list):
+        data = SpectrumList.read(data, format=format, data_label=data_label)
```

Comment:
```suggestion
        data = SpectrumList.read(data, format=format)
```

Passing `data_label` into `SpectrumList` doesn't do anything, right? It's not a keyword that's used by specutils, it's handled by the parser code later on.

Reply:
Fixed